packages/filecoin-wallet-provider/src/providers/local-node-provider.ts

```diff
@@ -0,0 +1,39 @@
+import LotusRpcEngine from '@glif/filecoin-rpc-client'
+import { LotusMessage, SignedLotusMessage } from '@glif/filecoin-message'
+import { WalletSubProvider } from '../wallet-sub-provider'
+
+export type LocalNodeProviderOptions = {
+  apiAddress: string
+  token?: string
+}
+
+export class LocalNodeProvider implements WalletSubProvider {
+  readonly apiAddress: string
+  readonly token?: string
+  readonly jsonRpcEngine: LotusRpcEngine
+
+  constructor(options: LocalNodeProviderOptions) {
+    this.apiAddress = options.apiAddress
+    this.token = options.token
+    this.jsonRpcEngine = new LotusRpcEngine({
+      apiAddress: options.apiAddress,
+      token: options.token,
+    })
+  }
+
+  newAccount(): Promise<string> {
```

Comment:
I think we can leave `newAccount` out - none of the other subproviders I use in the actual wallet need it, and `getAccounts` pretty much covers it. 

I had added that early on, following one of MetaMask's leads, but I dont see a reason we need it (unless you do)